src/objective-c/tests/PerfTests/PerfTests.m

```diff
@@ -265,37 +262,40 @@ - (void)testPingPongRPCWithV1API {
   }];
 }
 
-- (void)unaryRPCWithRequest:(RMTSimpleRequest *)request
-                numMessages:(int)numMessages
-                callOptions:(GRPCMutableCallOptions *)options {
-  const int kOutstandingRPCs = 10;
-  NSAssert(numMessages > kOutstandingRPCs, @"Number of RPCs must be > %d", kOutstandingRPCs);
+- (void)unaryRPCWithServices:(NSArray<RMTTestService *> *)services
+                     request:(RMTSimpleRequest *)request
+                 numMessages:(int)numMessages
```

Comment:
Since this method is making multiple calls, the following might make sense:
unaryRPCWithServices -> unaryRPCsWithServices
numMessages -> callsPerService   (note objc style guide discourages abbreviation)
numOutstanding -> maxOutstandingCalls